classifier/__init__.py

```diff
@@ -2982,6 +2992,8 @@ def audience(cls, identifier, name):
 
     @classmethod
     def target_age(cls, identifier, name):
+        if identifier == 'beginning reader':
+            return cls.range_tuple(5,8)
```

Comment:
This isn't what RBdigital means by "beginning reader" -- it's the industry standard seen elsewhere in this file.